csharp/tools/MauiModelTester/Utils.cs

```diff
@@ -61,7 +61,7 @@ internal static OrtValue CreateOrtValueFromTensorProto(Onnx.TensorProto tensorPr
             OrtValue ortValue = null;
 
             // special case for strings
-            if (tensorElementType == typeof(string))
+            if ((TensorElementType)tensorProto.DataType == TensorElementType.String)
```

Comment:
why does `GetElementType(TensorElementType.String)` return `typeof(byte)` and not `typeof(string)`?

Reply:
Hmmm. Good point. Not sure why it does. Will change it to return string.